ext/phar/phar_object.c

```diff
@@ -870,6 +867,7 @@ PHP_METHOD(Phar, webPhar)
 		code = phar_file_type(&PHAR_G(mime_types), entry, &mime_type);
 	}
 	phar_file_action(phar, info, mime_type, code, entry, entry_len, fname, pt, ru, ru_len);
+	efree(pt);
```

Comment:
I think I understand why this was done the way it was: the phar_file_action function may bail out in which case the memory of `pt` would be leaked.
However, I'm planning on getting rid of the bailouts in phar anyway, so this change is fine.
One question though: you said "requires unnecessary NULL checks" in the commit description but I didn't quite understand what you meant with that

Reply:
I am now trying to figure out what my brain at 1 AM was thinking, because yeah this does sound nonsensical.